jupyter-extension/jupyterlab_pachyderm/handlers.py

```diff
@@ -630,6 +633,7 @@ def setup_handlers(
 
     _handlers = [
         ("/repos", ReposHandler),
+        ("/mount", MountPfsRepoHandler),
```

Comment:
Is this mounted branch only applicable to the explore view? If so I think it makes sense to organize it outside of the top level to imply some containment

Reply:
Yep this is exclusive to the explore view. Agreed this sounds like a good suggestion.

Reply:
Added a `/explore` to this endpoints and renamed the handlers